app/scripts/deploy_service.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Deploys a Hivemind contract to localhost:8546
+ *
+ * Usage: node deploy.js <model_config file path> <deposit_amount>
+ */
+
+const chalk = require('chalk');
+const oasis = require('@oasislabs/client');
+const fs = require('fs');
+
+var multisig_deploy = oasis.workspace.Multisig.deploy(
+  ['b8b3666d8fea887d97ab54f571b8e5020c5c8b58', 'ff8c7955506c8f6ae9df7efbc3a26cc9105e1797'], 2,
+  { header: {confidential: false} }
+);
+
+multisig_deploy.then(async (multisig) => {
```

Comment:
why don't we use the async await syntax here?

Reply:
It isn't within an async call so await can't be used.